ImageGallery/src/org/sleuthkit/autopsy/imagegallery/datamodel/grouping/GroupManager.java

```diff
@@ -238,7 +243,8 @@ synchronized public void reset() {
         setGroupBy(DrawableAttribute.PATH);
         setSortOrder(SortOrder.ASCENDING);
         setDataSource(null);
-
+        resetLastUpdatedPathGroup();
```

Comment:
@bcarrier and @raman-bt, I don't think it is correct to call this here. The case is closed and the controller is shutting down. This should just set lastUpdatedPathGroup to null.